styles/feed-event.mcss

```diff
@@ -37,6 +37,10 @@ FeedEvent {
 
   div.meta {
     font-size: 100%
-    padding: 10px
+    padding: 10px 20px
+    opacity: 0.4
+    :hover {
+      opacity: 0.8
```

Comment:
Not a huge fan of the way the hover works on this.

Maybe it should be hover on "any" of the message to trigger? Maybe a short transition would help?

Maybe a colour should just be settled on that is a nice compromise that doesn't need a hover at all?

Interesting to note that in the case of replies to recent messages, this heading isn't all that important. It's just telling you why this item has been bumped to the top of your feed.

However in the case that this is an old item, it is actually telling you what the reply is for (it links to the full thread). This case is much more important. Maybe it should be handled slightly different? More like "View Full Thread".

Some thoughts.